### DISCUSSION TITLE
Jukebox should be able to set has_record directly

### DISCUSSION THREAD

**Discussion by TWME-TW** [PaperMC/Paper#10572] [Ideas]
### Is your feature request related to a problem?

I can set `has_record` to true directly through debugstick, but not through `org.bukkit.block.data.type.Jukebox`
. Obviously We need a method to set this value directly.

### Describe the solution you'd like.

Add `setHasRecord(boolean)`

### Describe alternatives you've considered.

Use `Server.createBlockData(java.lang.String)` to create a Jukebox with a specific `has_record` value

**Comment by Lulu13022002** [comment]
Resolved in 1.21.5 with the new generator
